app/models/inventory.rb

```diff
@@ -48,4 +50,10 @@ def owner?(user:)
   def member?(user:)
     self.members.where(user: user).exists?
   end
+
+  private
+  def add_facilitator_owner
+    return unless owner
+    facilitators.create(user: owner)
```

Comment:
members are allowed to have `invited_at` as nil, for example, when you're added as a district user.

We don't need to check if the owner already exists as facilitators since this method is only called with `after_create`.